topic_monitor/topic_monitor/launch_files/launch_reliability_demo.py

```diff
@@ -25,6 +26,10 @@ def main():
     package = 'topic_monitor'
     executable = get_executable_path(package_name=package, executable_name='data_publisher')
 
+    # Strip the logger name from the message format in favor of the shorter executable name
+    os.environ['RCUTILS_CONSOLE_OUTPUT_FORMAT'] = '[{severity}] {message}'
```

Comment:
Does this modified environment need to be passed to the launched processes ? (maybe it's not necessary as the same environment is used for the launchfile and each launched process). Otherwise there's an example of custom environment passed to launched processes here:
https://github.com/ros2/system_tests/blob/a4e81e348f7d2b25076f2b11dfde0f073e3850de/test_communication/test/test_publisher_subscriber.py.in#L28-L32

Reply:
yeah any children processes will inherit this environment automatically

Reply:
:+1: